gems/smithy/spec/spec_helper.rb

```diff
@@ -23,18 +23,26 @@ class << self
     def generate(modules, type, options = {})
       model = load_model(modules, options)
       plan = create_plan(modules, model, type, options)
+      sdk_dir = plan.options[:destination_root]
       smith(plan)
 
-      $LOAD_PATH << ("#{plan.options[:destination_root]}/lib")
+      $LOAD_PATH << ("#{sdk_dir}/lib")
       require "#{plan.options[:gem_name]}#{type == :types ? '-types' : ''}"
-      plan.options[:destination_root]
+
+      setup_rbs_spytest(modules, sdk_dir) if options.fetch(:rbs_test, ENV.fetch('SMITHY_RUBY_RBS_TEST', nil))
```

Comment:
I think the environment variable can be the flag. I don't think we need rbs_test as an actual option (we would never call this in specs right?)